File: BikeShop.Backend/BikeShop.Service/BikeShop.Service.Domain/Entities/ServiceWork.cs
using System.Text.Json.Serialization;

namespace BikeShop.Service.Domain.Entities;

// Работа, которая применялась при ремонте
public class ServiceWork : BaseEntity
{
    public int WorkId { get; set; }
    public string Name { get; set; } = string.Empty;
    public string Description { get; set; } = string.Empty;
    public int Quantity { get; set; } = 0;
    public decimal Price { get; set; } = 0;
    public decimal Discount { get; set; } = 0;
    public decimal ComplicationPrice { get; set; } = 0;
    public decimal Total { get; set; } = 0;
    public Guid? UserId { get; set; }

    public int ServiceId { get; set; } = 0;
}

File: BikeShop.Frontend/webclient/src/entities/entities/Service/ServiceWork.ts
export interface ServiceWork {
    id: number,
    createdAt: string,
    updatedAt: string,
    enabled: true,
    workId: number,
    name: string,
    description: string,
    quantity: number,
    price: number,
    discount: number,
    total: number,
    userId: string,
    serviceId: number
}